#!/bin/sh /usr/share/dpatch/dpatch-run
## 04-alt-gr.dpatch by Fabian Greffrath <fabian@debian-unofficial.org>
##
## DP: Enable `Alt Gr' key.

@DPATCH@
diff -urNad rott-1.0+dfsg~/rt_in.c rott-1.0+dfsg/rt_in.c
--- rott-1.0+dfsg~/rt_in.c	2008-01-10 09:16:47.000000000 +0100
+++ rott-1.0+dfsg/rt_in.c	2008-01-10 09:16:47.000000000 +0100
@@ -1093,6 +1093,7 @@
     
     scancodes[SDLK_LALT]            = sc_Alt;
     scancodes[SDLK_RALT]            = sc_Alt;
+    scancodes[SDLK_MODE]            = sc_Alt;
     scancodes[SDLK_RCTRL]           = sc_Control;
     scancodes[SDLK_SPACE]           = sc_Space;
     scancodes[SDLK_CAPSLOCK]        = sc_CapsLock;
